src/main/java/dev/openfeature/javasdk/Value.java

```diff
@@ -0,0 +1,206 @@
+package dev.openfeature.javasdk;
+
+import java.time.ZonedDateTime;
+import java.time.format.DateTimeFormatter;
+import java.util.List;
+
+import lombok.EqualsAndHashCode;
+import lombok.ToString;
+
+@ToString
+@EqualsAndHashCode
+@SuppressWarnings("PMD.BeanMembersShouldSerialize")
+public class Value {
```

Comment:
```suggestion
public class StructureValue {
```
?

Reply:
Not sure I agree with this one. I would be OK with `Node`, but I think `Value` is better.

Adding "Structure" seems wrong, since the value can be a primitive.